packages/test-utils/package.json

```diff
@@ -11,7 +11,8 @@
     "dist"
   ],
   "scripts": {
-    "build": "tsc -b"
+    "build": "tsc -b",
+    "dev": "pnpm run build --watch --preserveWatchOutput"
```

Comment:
I'm not a fan of this either as it feels like 90% of the time you don't need it.

What if instead we had a `test:dev` for the `runtime` which ran `pnpm run --filter=test-utils build --watch --preserveWatchOutput`?

Reply:
Actually this package should be consumed as source files by other packages. This package shouldn't have build step at all.

Reply:
Oh I agree! That would be even better! :100: 

Do you think you could implement this in this PR?

Reply:
Applied in https://github.com/stackblitz/tutorialkit/pull/141/commits/483e068cdf4202e47b62de2b3b9fbda3fd376fe4 - what do you think?

Reply:
Oh this looks really great! :star_struck: Good job on this!! :muscle: